packages/ocs/modals/storage-pool/delete-storage-pool-modal.tsx

```diff
@@ -207,18 +264,18 @@ const DeleteBlockPoolModal: React.FC<DeleteBlockPoolModalProps> = (props) => {
         </>
       ) : (
         <StatusBox
-          loaded={pvcLoaded && scLoaded}
-          loadError={pvcLoadError ?? scLoadError}
-          label={t('BlockPool Delete Modal')}
+          loaded={isLoaded}
+          loadError={loadError}
+          label={t('Storage Pool Delete Modal')}
         />
       )}
     </Modal>
   );
 };
 
 type DeleteBlockPoolModalProps = CommonModalProps<{
```

Comment:
same for name of the FC as well...
```suggestion
type DeleteStoragePoolModalProps = CommonModalProps<{
```

Reply:
Done.